### DISCUSSION TITLE
Data exchange through XML - unable to locate the product type

### DISCUSSION THREAD

**Discussion by ringorainbow** [amzn/selling-partner-api-models#3406] [General]
We would like to upload product in

Category: Clothing, Shoes & Jewelry Women Accessories Scarves & Wraps
Product Type: SCARF

in XSD
https://sellercentral.amazon.com/help/hub/reference/G1611

Category XSDs
Clothing, Accessories & Luggage
https://images-na.ssl-images-amazon.com/images/G/01/rainier/help/xsd/release_1_9/ClothingAccessories.xsd

or
ProductClothing.xsd
https://images-na.ssl-images-amazon.com/images/G/01/rainier/help/xsd/release_1_9/ProductClothing.xsd

We are unable to locate the ClothingType or ProductType "SCARF" in the xsd file and therefore we are unable to upload the product without the ClothingType or ProductType

Can anyone help?

**Comment by walekazm** [comment]
i can help

**Comment by Judae** [comment]
How

Jude Nwawere
